resources/idlharness.js

```diff
@@ -3010,7 +3018,143 @@ function IdlTypedef(obj)
 }
 //@}
 
-IdlTypedef.prototype = Object.create(IdlObject.prototype);
+IdlNamespace.prototype = Object.create(IdlObject.prototype);
+
+/// IdlNamespace ///
+function IdlNamespace(obj)
+//@{
+{
+    this.name = obj.name;
+    this.extAttrs = obj.extAttrs;
+    this.untested = obj.untested;
+    /** A back-reference to our IdlArray. */
+    this.array = obj.array;
+
+    /** An array of IdlInterfaceMembers. */
+    this.members = obj.members.map(m => new IdlInterfaceMember(m));
+    if (this.has_extended_attribute("Unforgeable")) {
+        this.members.forEach(m => { m.isUnforgeable = true; });
+    }
+}
+//@}
+
+IdlNamespace.prototype.do_member_operation_asserts = function (memberHolderObject, member, a_test)
+//@{
+{
+    var done = a_test.done.bind(a_test);
+    var operationUnforgeable = member.has_extended_attribute("Unforgeable");
```

Comment:
Should this check the `isUnforgeable` property instead?

Reply:
Done.